use-timescale/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -31,84 +31,154 @@ For more information, see the documentation about
+*   You can use an `INNER`, `LEFT` and `LATERAL` joins, no other join type is supported.
 *   Joins on the materialized hypertable of a continuous aggregate are not supported.
 *   Hierarchical continuous aggregates can be created on top of a continuous
     aggregate with a `JOIN` clause, but cannot themselves have a `JOIN` clauses.
 
-This section includes some examples of `JOIN` conditions that work with
-continuous aggregates. For these to work, either `table_1` or `table_2` must be
-a hypertable. It does not matter which is the hypertable and which is a standard
-PostgreSQL table.
-
-`INNER JOIN` on a single equality condition, using the `ON` clause:
+### JOIN examples
 
-```sql
-CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
-SELECT ...
-FROM table_1 t1
-JOIN table_2 t2 ON t1.t2_id = t2.id
-GROUP BY ...
-```
-
-`INNER JOIN` on a single equality condition, using the `ON` clause, with a further
-condition added in the `WHERE` clause:
+Given the following schema:
 
 ```sql
-CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
-SELECT ...
-FROM table_1 t1
-JOIN table_2 t2 ON t1.t2_id = t2.id
-WHERE t1.id IN (1, 2, 3, 4)
-GROUP BY ...
+CREATE TABLE locations (
+    id TEXT PRIMARY KEY,
+    name TEXT
+);
+
+CREATE TABLE devices (
+    id SERIAL PRIMARY KEY,
+    location_id TEXT,
+    name TEXT
+);
+
+CREATE TABLE conditions (
+    "time" TIMESTAMPTZ,
+    device_id INTEGER,
+    temperature FLOAT8
+);
+
+SELECT create_hypertable('conditions', by_range('time'));
 ```
 
-`INNER JOIN` on a single equality condition specified in `WHERE` clause, this is allowed but not recommended:
-
-```sql
-CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
-SELECT ...
-FROM table_1 t1, table_2 t2
-WHERE t1.t2_id = t2.id
-GROUP BY ...
-```
-
-These are examples of `JOIN` conditions won't work with continuous aggregates:
-An `INNER JOIN` on multiple equality conditions is not allowed.
-
-```sql
-CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
-SELECT ...
-FROM table_1 t1
-JOIN table_2 t2 ON t1.t2_id = t2.id AND t1.t2_id_2 = t2.id
-GROUP BY ...
-```
-
-A `JOIN` with a single equality condition specified in `WHERE` clause cannot be combined with further conditions in the `WHERE` clause.
-
-```sql
-CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
-SELECT ...
-FROM table_1 t1, table_2 t2
-WHERE t1.t2_id = t2.id
-AND t1.id IN (1, 2, 3, 4)
-GROUP BY ...
-```
+See the following `JOIN` examples on Continuous Aggregates:
+
+- `INNER JOIN` on a single equality condition, using the `ON` clause:
+
+    ```sql
+    CREATE MATERIALIZED VIEW conditions_by_day WITH (timescaledb.continuous) AS
+    SELECT time_bucket('1 day', time) AS bucket, devices.name, MIN(temperature), MAX(temperature)
+    FROM conditions
+    JOIN devices ON devices.id = conditions.device_id
+    GROUP BY bucket, devices.name
+    WITH NO DATA;
+    ```
+    TimescaleDB v2.15.x and higher. 
```

Comment:
```suggestion
    TimescaleDB v2.10.x and higher. 
```
Do u think we need to specify it on every example? I mean we have the table above. I guess we should only have a note for >= 2.16.x (new cagg join features)...

Reply:
Agreed, removed the 2.15 lines.